From 16ef749bfa0ce34129cc41175ea5c834f65be749 Mon Sep 17 00:00:00 2001
From: Koji Matsuoka <koji.matsuoka.xm@renesas.com>
Date: Mon, 27 Aug 2018 12:56:52 +0900
Subject: [PATCH 523/909] arm64: dts: r8a77995-draak: Add no use ddc option for
 VGA connector

commit 837b6af985e76e65351c04caffdd1936e5521c7c from
git://git.kernel.org/pub/scm/linux/kernel/git/horms/renesas-bsp.git

By setting this option, change to not use DDC (Display Data Channel)
on the VGA connector.

Signed-off-by: Koji Matsuoka <koji.matsuoka.xm@renesas.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/renesas/r8a77995-draak.dts | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/arch/arm64/boot/dts/renesas/r8a77995-draak.dts b/arch/arm64/boot/dts/renesas/r8a77995-draak.dts
index 0849fa66c74c..4e34a9ac0563 100644
--- a/arch/arm64/boot/dts/renesas/r8a77995-draak.dts
+++ b/arch/arm64/boot/dts/renesas/r8a77995-draak.dts
@@ -27,6 +27,8 @@
 	vga {
 		compatible = "vga-connector";
 
+		no-use-ddc;
+
 		port {
 			vga_in: endpoint {
 				remote-endpoint = <&adv7123_out>;
-- 
2.17.1

